src/bors/handlers/refresh.rs

```diff
@@ -105,132 +105,131 @@ fn elapsed_time(date: DateTime<Utc>) -> Duration {
-//                     assert_eq!(
-//                         tester
-//                             .db()
-//                             .get_running_builds(&default_repo_name())
-//                             .await
-//                             .unwrap()
-//                             .len(),
-//                         1
-//                     );
-//                     tester.refresh().await;
-//                 })
-//                 .await;
-//                 insta::assert_snapshot!(tester.get_comment().await?, @":boom: Test timed out");
-//                 assert_eq!(
-//                     tester
-//                         .db()
-//                         .get_running_builds(&default_repo_name())
-//                         .await
-//                         .unwrap()
-//                         .len(),
-//                     0
-//                 );
-//                 Ok(tester)
-//             })
-//             .await;
-//     }
-//
-//     #[sqlx::test]
-//     async fn refresh_cancel_workflow_after_timeout(pool: sqlx::PgPool) {
-//         let world = World::default();
-//         world.default_repo().lock().set_config(
-//             r#"
-// timeout = 3600
-// "#,
-//         );
-//         let world = BorsBuilder::new(pool)
-//             .world(world)
-//             .run_test(|mut tester| async move {
-//                 tester.post_comment("@bors try").await?;
-//                 tester.expect_comments(1).await;
-//                 tester
-//                     .workflow_event(WorkflowEvent::started(tester.try_branch()))
-//                     .await?;
-//                 WAIT_FOR_WORKFLOW_STARTED.sync().await;
-//
-//                 with_mocked_time(Duration::from_secs(4000), async {
-//                     tester.refresh().await;
-//                 })
-//                 .await;
-//                 tester.expect_comments(1).await;
-//                 Ok(tester)
-//             })
-//             .await;
-//         world.check_cancelled_workflows(default_repo_name(), &[1]);
-//     }
-//
-//     async fn with_mocked_time<Fut: Future<Output = ()>>(in_future: Duration, future: Fut) {
-//         // It is important to use this function only with a single threaded runtime,
-//         // otherwise the `MOCK_TIME` variable might get mixed up between different threads.
-//         assert_eq!(
-//             tokio::runtime::Handle::current().runtime_flavor(),
-//             RuntimeFlavor::CurrentThread
-//         );
-//         MOCK_TIME.with(|time| {
-//             *time.borrow_mut() = Some(Utc::now() + chrono::Duration::from_std(in_future).unwrap());
-//         });
-//         future.await;
-//         MOCK_TIME.with(|time| {
-//             *time.borrow_mut() = None;
-//         });
-//     }
-// }
+#[cfg(test)]
+mod tests {
+    use crate::bors::handlers::refresh::MOCK_TIME;
+    use crate::bors::handlers::WAIT_FOR_WORKFLOW_STARTED;
+    use crate::bors::WAIT_FOR_REFRESH;
```

Comment:
I don't think it's necessary to wait for refresh event in any of the tests?

Reply:
Well, we *do* need to wait, but the waiting is handled inside the `refresh` call, so this import is indeed unused. Good catch, thanks!

Reply:
Ah yeah, that's what I meant. Sorry for not expressing myself clearly😅